internal/pkg/scorecard/helpers/versions.go

```diff
@@ -0,0 +1,58 @@
+// Copyright 2019 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package schelpers
+
+import (
+	"fmt"
+	"github.com/spf13/viper"
+	"strings"
+)
+
+const V1alpha1 = "v1alpha1"
+const V1alpha2 = "v1alpha2"
+const DefaultScorecardVersion = V1alpha1
+const LatestScorecardVersion = V1alpha2
+const VersionOpt = "version"
+
+var ScorecardVersions = []string{DefaultScorecardVersion, LatestScorecardVersion}
+
+func ValidateVersion(version string) error {
+	validVersion := false
+	for _, a := range ScorecardVersions {
+		if a == version {
+			validVersion = true
```

Comment:
You can just `return nil` here, and `return fmt.Errorf("invalid ...")` at the function's end.

Reply:
thanks, good catch.